x-pack/plugins/siem/public/components/open_timeline/delete_timeline_modal/delete_timeline_modal.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { pure } from 'recompose';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import * as i18n from '../translations';
+
+const DeleteTimelineModalContainer = styled(EuiPanel)<{ width: number }>`
+  width: ${({ width }) => `${width}px`};
+  user-select: none;
+`;
+
+const CancelFlexItem = styled(EuiFlexItem)`
+  margin-right: 12px;
+`;
+
+interface Props {
+  title?: string | null;
+  onDelete: () => void;
+  toggleShowModal: () => void;
+}
+
+export const DELETE_TIMELINE_MODAL_WIDTH = 600; // px
+
+/**
+ * Renders a modal that confirms deletion of a timeline
+ */
+export const DeleteTimelineModal = pure<Props>(({ title, toggleShowModal, onDelete }) => (
+  <DeleteTimelineModalContainer paddingSize="l" width={DELETE_TIMELINE_MODAL_WIDTH}>
+    <EuiTitle size="m">
+      <h2>
+        <FormattedMessage
+          id="xpack.siem.open.timeline.deleteTimelineModalTitle"
+          data-test-subj="title"
+          defaultMessage="Delete `{title}`?"
```

Comment:
Are the backticks intentional? Or did you want `'`'s?
![image](https://user-images.githubusercontent.com/2946766/57031158-281a6c00-6c04-11e9-8721-775c8140b684.png)

Reply:
Per the screenshot below, the backticks are intentional because they are included in the wireframes:

![modal-wireframe](https://user-images.githubusercontent.com/4459398/57042407-c9fc8180-6c21-11e9-837c-2e0db3912247.png)